primer/test/Tests/Eval.hs

```diff
@@ -936,7 +936,7 @@ unit_findNodeByID_capture_type =
             , Just (1, LetTyBind _) <- lookupCaptured "x" locals
             , Just (5, LetTyBind _) <- lookupCaptured "z" locals ->
                 pure ()
-          Just (_, Right _) -> assertFailure "Expected lettype binding of 'x' and the tlet binding of 'z' to be reported as captured-if-inlined"
+          Just (_, Right _) -> assertFailure "Expected lettype binding of 'x' and the tlet binding of 'z' to be reported as captured-if-inlined" -- TODO: can probably remove all the "captured-if-inlined" stuff as don't do inlining like that now
```

Comment:
This is referring to one of the simplifications that I should do & document

Reply:
Is this done now?

Reply:
Same goes for the other three discussion threads mentioning "simplifications".

Reply:
Yes, in the later commits (this one is in "change Eval.Cxt to match push-down lets")